troubleshooting.adoc

```diff
@@ -72,12 +72,12 @@ system:
 sudo apt-get install build-essential uuid-dev zlib1g-dev python-dev
 --------------------------------------------------------------------
 
-Download protobuf version 2.5.0 or later from link:http://code.google.com/p/protobuf/[protobuf]
+Download protobuf version 3.0.0 or later from link:http://code.google.com/p/protobuf/[protobuf]
```

Comment:
you actually only need the compiler, so you can just go to the releases page and download the relevant compiler package for your OS.